bindings/java/src/org/sleuthkit/datamodel/timeline/TimelineEvent.java

```diff
@@ -88,16 +86,17 @@
 	 */
 	public TimelineEvent(long eventID, long dataSourceObjID, long fileObjID, Long artifactID,
 			long time, EventType type,
-			EventDescription descriptions,
+			String fullDescription,
+			String medDescription,
+			String shortDescription,
 			boolean hashHit, boolean tagged) {
 		this.eventID = eventID;
 		this.dataSourceObjID = dataSourceObjID;
 		this.fileObjID = fileObjID;
 		this.artifactID = Long.valueOf(0).equals(artifactID) ? null : artifactID;
 		this.time = time;
 		this.type = type;
-		this.descriptions = descriptions;
-
+		this.descriptions = type.parseDescription(fullDescription, medDescription, shortDescription);
```

Comment:
we need to take the raw descriptions from the DB and parse out the levels of detail (for web and file system events).